app/javascript/components/admin/site_settings/Appearance.jsx

```diff
@@ -1,7 +1,36 @@
+/* eslint-disable react/jsx-props-no-spreading */
+
 import React from 'react';
+import { Form } from 'react-bootstrap';
+import Button from 'react-bootstrap/Button';
+import { useForm } from 'react-hook-form';
+import useUpdateSiteSetting
+  from '../../../hooks/mutations/admins/site_settings/useUpdateSiteSetting';
+import useSiteSettings from '../../../hooks/queries/admin/site_settings/useSiteSettings';
 
 export default function Appearance() {
+  const { register, handleSubmit } = useForm();
+  const updateSiteSetting = useUpdateSiteSetting('PrimaryColor');
+  const { data: siteSettings } = useSiteSettings();
```

Comment:
I'm seeing two different approaches with this. Sometimes we just do 
`const { siteSettings } = useSiteSettings();` and then do `siteSettings.data.PrimaryColor' for example.

What should be the way that we stick to ?
@scouillard @farhatahmad @KH-Amir-TN